registry/vk.xml

```diff
@@ -6217,10 +6217,10 @@ typedef void <name>CAMetalLayer</name>;
         <type category="include" name="vk_video/vulkan_video_codec_h265std_encode.h">#include "vk_video/vulkan_video_codec_h265std_encode.h"</type>
         <type requires="vk_video/vulkan_video_codec_h265std_encode.h" name="StdVideoEncodeH265PictureInfoFlags"/>
         <type requires="vk_video/vulkan_video_codec_h265std_encode.h" name="StdVideoEncodeH265PictureInfo"/>
-        <type requires="vk_video/vulkan_video_codec_h265std_encode.h" name="StdVideoEncodeH265SliceHeader"/>
+        <type requires="vk_video/vulkan_video_codec_h265std_encode.h" name="StdVideoEncodeH265SliceSegmentHeader"/>
```

Comment:
@oddhack it doesn't look like this header file was updated, it still provides `StdVideoEncodeH265SliceHeader` but not `StdVideoEncodeH265SliceSegmentHeader`, rendering `vk.xml` broken once again :(